.ocamlformat

```diff
@@ -1,3 +1,3 @@
-version=0.24.1
+version=0.25.1
```

Comment:
Just for some context: the `ocamlformat.0.25` version was released a couple of weeks ago, and we haven't upgraded it yet. You've probably seen an error when running `dune @build` pointing out that you have `ocamlformat.0.25.1` installed, but `ocamlformat.0.24.1` specified in the `.ocamlformat` file. Your solution was to update the `.ocamlformat`-file to make sure the two coincide. Is that right?

If so, the cleaner solution would have been to downgrade your installed `ocamlformat` version to `0.24.1`. That way, we would keep a clean git history: one commit/PR adding the two new scales; and, possibly, later another commit/PR upgrading `ocamlformat` and re-formatting the code base.

All this explanation is just to give you some context. In this project, we don't need to be too pedantic about the git history and can keep your approach mixing the two things in one. @Engil , let me know if you think otherwise!

Reply:
Thanks for explaining the issue @pitag-ha !
I have downgraded back to 0.24.1, and will open separate PR for upgrading ocamlformat in the repo!

Reply:
And yes, my plan was to concide the ocamlformat versions :)